fgd/brush/trigger/trigger_momentum_timer_stage.fgd

```diff
@@ -1,6 +1,11 @@
-@SolidClass base(ZoneTrigger)
-	appliesto(MOMENTUM)
-= trigger_momentum_timer_stage : "Starting trigger for each stage of a map. trigger_momentum_timer_start is automatically stage 1!" 
+@SolidClass appliesto(MOMENTUM)
+= trigger_momentum_timer_stage : "Starting zone for each stage of a map. trigger_momentum_timer_start is automatically stage 1!" 
 	[
-	teleport_destination(target_destination) : "Remote Destination" : : "The entity specifying the point to which the player should be teleported when restarting a stage."
+	targetname(target_source) : "Name" : : "The name of this zone."
+	filtername(string) : "Filter Name" : : "filter_activator entity to use for this zone."
+	stage_number(integer) : "Stage Number" : 2 : "The stage number of this zone. NOTE: 1 is reserved for start! 2+ are free for your stage zones, so start at 2!"
+	checkpoints_required(boolean) : "Checkpoints Required" : 1 : "Are checkpoints required to complete this track?"
```

Comment:
This is only relevant when a stage has checkpoints within it. Assuming we want to support defining those in Hammer, trigger_momentum_timer_checkpoint needs a stage number field in addition to the checkpoint number.

Also maybe clarify:
```suggestion
	checkpoints_required(boolean) : "Checkpoints Required" : 1 : "Is the player required to activate all checkpoints (in order) before progressing to the next stage? Otherwise, checkpoint zones may be skipped for this stage."
```